src/h5web/visualizations/line/DataCurve.tsx

```diff
@@ -1,40 +1,35 @@
-import React, { Suspense } from 'react';
-import { useUpdate } from 'react-three-fiber';
+import React, { Suspense, useMemo } from 'react';
 import { BufferGeometry } from 'three';
+import { Line } from 'react-three-fiber/components';
 import { useDataPoints } from './hooks';
 import { useLineConfig } from './config';
-import { Glyph } from './models';
-import DataCurveMaterial from './DataCurveMaterial';
+import { CurveType } from './models';
+import DataGlyphs from './DataGlyphs';
 
 function DataCurve(): JSX.Element {
   const points = useDataPoints();
-  const ref = useUpdate(
-    (geometry: BufferGeometry) => {
-      if (points) {
-        geometry.setFromPoints(points);
-      }
-    },
-    [points]
-  );
-  const glyph = useLineConfig(state => state.glyph);
-  const color = '#1b998b';
+  const dataGeometry = useMemo(() => {
+    const geometry = new BufferGeometry();
+    if (points) {
+      geometry.setFromPoints(points);
+    }
+    return geometry;
+  }, [points]);
```

Comment:
This allows to compute the geometry only once and use it for both the line and point materials.

Reply:
So we recreate the geometry when the points change. Alright, not a big deal since it only happens when switching to a log scale. 👍